src/engine/openspaceengine.cpp

```diff
@@ -176,6 +177,13 @@ OpenSpaceEngine::OpenSpaceEngine(
     TransformationManager::initialize();
 }
 
+OpenSpaceEngine::~OpenSpaceEngine() {
+    // We know that we own all of the properties in the virtual namespace
+    for (properties::Property* prop : _virtualPropertyNamespace->propertiesRecursive()) {
+        delete prop;
```

Comment:
I guess we should consider creating unique pointers to props later on, and pass ownership to the propery owners. Probably not in this PR though.